src/main/resources/application.properties

```diff
@@ -0,0 +1,12 @@
+#DataSource properties
+spring.datasource.url=jdbc:postgresql://localhost:5432/training
+spring.datasource.username=dev_training
+spring.datasource.password=wolox1189
```

Comment:
For the training purposes this is fine, but keep in mind that in projects the credentials shouldn't be pushed 🙂